wear/src/main/res/layout/activity_contact_events.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:tools="http://schemas.android.com/tools"
+  android:layout_width="match_parent"
+  android:layout_height="match_parent"
+  tools:context="com.alexstyl.specialdates.ContactEventsActivity"
+  tools:deviceIds="wear_round">
+
+  <LinearLayout
+    android:id="@+id/contact_events_event_container"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content"
+    android:layout_centerInParent="true"
+    android:gravity="center"
+    android:orientation="vertical">
+
+    <TextView
+      android:id="@+id/contact_events_date_text"
+      style="@style/ContactEvents.Label.Date"
```

Comment:
Can you please include the `drawableTop`, `drawablePadding` and `gravity` as well in the Style as well please?